.circleci/config.yml

```diff
@@ -0,0 +1,157 @@
+---
+# Python CircleCI 2.0 configuration file
+#
+# INFO: Check CircleCI documentation for more details
+#       https://circleci.com/docs/2.0/language-python/
+
+version: 2.1
+
+executors:
+  standard:
+    docker:
+      - image: circleci/python:3.9
+        environment:
+          PIPENV_VENV_IN_PROJECT: true
+    working_directory: "~/lib"
+
+# -----------------
+# Reusable commands
+# -----------------
+
+commands:
+  configure_github:
+    steps:
+      - run:
+          name: Configure Github access
+          command: |
+            git config --global --unset url.ssh://git@github.com.insteadof;\
+            echo https://${GITHUB_TOKEN}:@github.com > ~/.git-credentials
+            git config --global credential.helper store
+
+  checkout_source:
+    steps:
+      - restore_cache:
+          keys:
+            - source-{{ .Branch }}-{{ .Revision }}
+            - source-{{ .Branch }}-
+            - source-
+      - checkout
+      - save_cache:
+          key: source-{{ .Branch }}-{{ .Revision }}
+          paths:
+            - "./.git"
+
+# todo: remove unixodbc when tentaclio 1.0 is released
+  install_unixodbc:
+    steps:
+      - run:
+          name: Instal UnixODBC
+          command: |
+            sudo apt-get install unixodbc-dev -y
+
```

Comment:
prolly not needed in this repo

Reply:
Currently needed as relies on latest tentaclio which requires due to pyodbc, will drop once v1 is out!